examples/led.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * led.c - Using GPIO to control the LED on/off
+ */
+
+#include <linux/cdev.h>
+#include <linux/delay.h>
+#include <linux/device.h>
+#include <linux/fs.h>
+#include <linux/gpio.h>
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/printk.h>
+#include <linux/types.h>
+#include <linux/uaccess.h>
+#include <linux/version.h>
+
+#include <asm/errno.h>
+
+#define SUCCESS 0
+#define DEVICE_NAME "gpio_led"
+
+/* The major device number. We can not rely on dynamic registration
+ * any more.
+ */
+#define MAJOR_NUM 100
```

Comment:
Explain why.

Reply:
I saw this comment in chardev.h, and I consider it better to explicitly register the device number if possible.